api/bindings.h

```diff
@@ -140,12 +140,15 @@ Buffer handle(cache_t *cache,
               GoApi api,
               GoQuerier querier,
               uint64_t gas_limit,
-              uint32_t memory_limit,
               bool print_debug,
               uint64_t *gas_used,
               Buffer *err);
 
-cache_t *init_cache(Buffer data_dir, Buffer supported_features, uint32_t cache_size, Buffer *err);
+cache_t *init_cache(Buffer data_dir,
+                    Buffer supported_features,
+                    uint32_t cache_size,
+                    uint32_t instance_memory_limit,
```

Comment:
Ah, memory limit is here now, not in init.
I was wondering about the changes above

Reply:
Yeah, this is the trade-off we had to make in order to avoid Module serialization/deserialization: all instances have the same memory limit. It cannot be set to different values for different calls. But we did not use that anyways.

Reply:
Seems fair, we just have a global in wasmd